pkg/action/list.go

```diff
@@ -322,3 +335,15 @@ func (l *List) SetStateMask() {
 
 	l.StateMask = state
 }
+
+// Create a clone of the config with a new namespace
+func (cfg *Configuration) CloneWithNewNamespace(namespace string) (*Configuration, error) {
```

Comment:
I didn't think of the fact that action.go and list.go are in the same package. CloneWithNewNamespace could be changed to private cloneWithNewNamespace